External/FEXCore/Source/Utils/FileLoading.cpp

```diff
@@ -7,6 +9,7 @@
 #include <unistd.h>
 
 namespace FEXCore::FileLoading {
+// TODO: Delete once all uses of std::vector LoadFile is removed.
```

Comment:
Are there still uses of this remaining?

Generally, `[[deprecated]]` would be a good use here instead of a loose comment.

Reply:
Yes, there are still uses of this. It's getting deleted in a follow-up PR soon after this one.